langchain/schema.py

```diff
@@ -244,7 +307,7 @@ def flatten(self) -> List[LLMResult]:
                 )
             else:
                 if self.llm_output is not None:
-                    llm_output = self.llm_output.copy()
+                    llm_output = deepcopy(self.llm_output)
```

Comment:
this is intentional? why?

Reply:
test if so?

Reply:
couldn't think of a good reason we'd want to potentially share mutable objects across the flattened generations, assumed was mistake to not deep copy in first place. are you saying it's intentional to do shallow copy? don't love sharing dicts across objects, find it confusing